### PR TITLE
Make cookie httponly = false.

### PR DESCRIPTION
**Pull request by cmfcmf**
| Q                 | A
| ----------------- | ---
| Bug fix?          | yes
| New feature?      | no
| BC breaks?        | no
| Deprecations?     | no
| Tests pass?       | ---
| Fixed tickets     | ---
| Refs tickets      | ---
| License           | MIT
| Doc PR            | ---
| Changelog updated | no

### CODE REVIEW

**Comment by cmfcmf** [comment]
I would be glad if @craigh could write a test for this? :pray:

**Comment by cmfcmf** [comment]
> I would be glad if @craigh could write a test for this? :pray: 
Whoops, wrong ticket.

File: src/app/config/config.yml [hunk 1]
```diff
@@ -26,6 +26,7 @@ framework:
         name: %zikula.session.name%
         handler_id: session.handler.legacy
         storage_id: session.storage.legacy
+        cookie_httponly: false
```
Comment (cmfcmf) [inline src/app/config/config.yml]: If this is set to false, the session id cannot be retrieved in JavaScript via `document.cookie`. This is currently required however to make our AJAX requests work.

Comment (craigh) [inline src/app/config/config.yml]: I don't understand. you are setting it to false, but saying that it doesn't work if set to false?

**Comment by craigh** [comment]
is this related at all to https://github.com/zikula/core/issues/2672 ?

**Comment by cmfcmf** [comment]
> is this related at all to #2672 ?

I don't know. I'm not working in a multilingual setup at the moment. With this change `$this->checkAjaxToken()` works again.

**Comment by craigh** [comment]
> `$this->checkAjaxToken()` works again

:+1: :tada:

File: src/app/config/config.yml (hunk 1 above)
Comment (cmfcmf) [inline src/app/config/config.yml]: I mixed up true and false. Updated the comment.

**Pull request merged**